commands.py

```diff
@@ -702,7 +703,10 @@ async def shut(self, ctx):
                 change = "disabled messaging permissions"
                 await role.edit(permissions=discord.Permissions(permissions=104187456))
 
-        await ctx.send(change)
+        if change:
+            await ctx.send(change)
+        else:
+            await ctx.send("No messaging permissions changed")
```

Comment:
this else case is impossible. the states only flip between one or the other

Reply:
Yea, that would be an undefined state if it ever got there. It is technically possible (if someone really broke the Discord permissions). Just setting change to "" at the start and removing the if would probably be fine to at least ensure it does not crash if it did get to that situation somehow.